backend/src/main/resources/db/migration/V1__initializing_data.sql

```diff
@@ -46,8 +98,84 @@ CREATE TABLE IF NOT EXISTS nrfc.submission_detail (
 	last_name                 	varchar(100)    null,
 	client_type_code          	varchar(1)    	not null,
 	date_of_birth				date 			null,
-	client_comment				varchar(5000)	null,
-	CONSTRAINT submission_detail_id_pk PRIMARY KEY (submission_detail_id),
-	CONSTRAINT submission_detail_client_type_code_fk FOREIGN KEY (client_type_code) REFERENCES nrfc.client_type_code(client_type_code),
-    CONSTRAINT submission_detail_submission_id_fk FOREIGN KEY (submission_id) REFERENCES nrfc.submission(submission_id)
+    doing_business_as_ind       varchar(1)      not null default 'N',
+    doing_business_as_name      varchar(100)    null,
+	constraint submission_detail_id_pk primary key (submission_detail_id),
+	constraint ssubmission_id_fk foreign key (submission_id) references nrfc.submission(submission_id),
+    constraint submission_detail_client_type_code_fk foreign key (client_type_code) references nrfc.client_type_code(client_type_code),
+    constraint submission_detail_submission_id_fk foreign key (submission_id) references nrfc.submission(submission_id)
+);
+
+create table if not exists nrfc.submission_location (
+    submission_location_id      integer			not null,
+	submission_id               integer			not null,
+    street_address              varchar(50)    	not null,    
+    country_code                varchar(2)      not null,
+    province_code               varchar(2)      not null,
+    postal_code                 varchar(10)     not null,
+    is_billing_address_same_ind varchar(1)      not null default 'N',
+    constraint submission_location_id_pk primary key (submission_location_id),
+    constraint ssubmission_id_fk foreign key (submission_id) references nrfc.submission(submission_id),
+    constraint submission_location_country_code_fk foreign key (country_code) references nrfc.country_code(country_code),
+    constraint submission_location_province_code_fk foreign key (province_code) references nrfc.province_code(province_code)
+);
+
+create table if not exists nrfc.submission_contact (
+    submission_contact_id       integer			not null,
+	submission_id               integer			not null
+    //TODO
```

Comment:
This line is an illegal line. SQL comments on Postgres are usually single lined comments like `--` or multi-line like ` /* */`

Reply:
Sorry. Thanks for checking